homeassistant/components/foscam/camera.py

```diff
@@ -67,6 +68,21 @@ def camera_image(self):
 
         return response
 
+    @property
+    def supported_features(self):
+        """Return supported features."""
+        return SUPPORT_STREAM
+
+    @property
+    def stream_source(self):
+        """Return the stream source."""
+        return 'rtsp://%(username)s:%(password)s@%(host)s:%(port)s/videoMain' \
+            % dict(
```

Comment:
Please use `"string value".format(self._username)` to format strings.

Reply:
👍